cockroachcloud/export-metrics.md

```diff
@@ -0,0 +1,248 @@
+
+1. Copy the [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the `CockroachCloudMetricsExportRole` role found under **Summary**, which is needed for the next step.
+
+1. Issue the following Cloud API command to enable metrics export for your {{ site.data.products.dedicated }} cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request POST \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport \
+      --header "Authorization: Bearer {secret_key}" \
+      --data '{"cloudwatch":{"target_region": "{aws_region}}", "role_arn": "arn:aws:iam::{role_arn}:role/CockroachCloudMetricsExportRole", "log_group_name": "{log_group_name}"}}'
+    ~~~
+
+    Where:
+    - `{cluster_id}` is your {{ site.data.products.dedicated }} cluster ID as determined in step 2.
+    - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
+    - `{aws_region}` is your AWS region, like `us-east-1`.
+    - `{role_arn}` is the ARN for the `CockroachCloudMetricsExportRole` role you copied in step 7. If you used a different role name there, be sure to use your role name in place of `CockroachCloudMetricsExportRole` in the above command.
+    - `{log_group_name}` is the target AWS CloudWatch log group you created in step 1.
+
+    Specifying an AWS region (to `{aws_region}`) that you do not have a cluster in, or a region that only partially covers your cluster's nodes will result in missing metrics.
+
+1. Depending on the size of your cluster and how many regions it spans, the configuration may take a moment. You can monitor the ongoing status of the configuration using the following Cloud API command:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport \
+      --header "Authorization: Bearer {secret_key}"
+    ~~~
+
+    Run the command periodically until the command returns a status of `ENABLED`, at which point the configuration across all nodes is complete, and metrics will begin appearing in CloudWatch under the log group you created in step 1. Since the configuration is applied to cluster nodes in a rolling fashion, you may see some metrics appear even before the `GET` command returns an `ENABLED` status.
+
+1. Once metrics export has been enabled, you can access metrics from your {{ site.data.products.dedicated }} cluster directly in [AWS CloudWatch](https://console.aws.amazon.com/cloudwatch/home).
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="datadog-metrics-export">
+
+Perform the following steps to enable metrics export from your {{ site.data.products.dedicated }} cluster to Datadog.
```

Comment:
Suggestion: it might be helpful to add a sentence or two telling the user that they can also setup the Datadog integration through our UI and linking them to [https://www.cockroachlabs.com/docs/cockroachcloud/monitoring-page.html#monitor-cockroachdb-dedicated-with-datadog](https://www.cockroachlabs.com/docs/cockroachcloud/monitoring-page.html#monitor-cockroachdb-dedicated-with-datadog) for the readers' convenience